src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$tablet-min-width: 576px;
+$desktop-min-width: 1024px;
+
+$effectDuration: 0.3s;
```

Comment:
When naming variables, a single writing style should be followed (`$effect-duration`, `$font-family`).